tools/loader/ImageClassifier.cpp

```diff
@@ -279,6 +284,25 @@ static int processAndPrintResultsImpl(Tensor *SMT,
   const size_t numLabels = SMT->dims()[1];
   std::vector<size_t> sliceOffset(SMT->dims().size(), 0);
 
+  // Parse provided, if any, category indices.
+  std::vector<signed> matchingIndices(inputImageFilenames.size(), -1);
+  if (!expectedMatchingIndices.empty()) {
+		llvm::SmallVector<llvm::StringRef, 1> parts;
+    llvm::StringRef(expectedMatchingIndices).split(parts, ',', -1, false);
+    if (parts.size() != inputImageFilenames.size()) {
+      llvm::outs() << "Number of matching indices: " << parts.size()
+                   << " doesn't match the number of files: "
+                   << inputImageFilenames.size() << "\n";
+    } else {
+      for (unsigned i = 0; i < parts.size(); i++) {
+        if (parts[i].getAsInteger(10, matchingIndices[i])) { // true means error
```

Comment:
Please move the comment above the line and punctuate it. 

https://releases.llvm.org/7.0.0/docs/CodingStandards.html#comment-formatting

Reply:
Fixed in the last commit